### ISSUE TITLE
report_histogram does not show "horizontal" orientation 

### ISSUE THREAD

**Issue by ohad-pipl** [allegroai/clearml#699]
I am trying to create a histogram using `report_histogram` which will be displayed in the UI with horizontal orientation (the bars are on the y axis and the values are on the x axis). 

I was suggested in the community slack channel to use the `extra_layout` parameter of the function to pass the orientation as described in this link: [https://plotly.com/javascript/reference/bar/#bar-orientation](url)
However this still yielded the "vertical" orientation.

Example code:

```
clearml_task.logger.report_histogram(
                    "test histogram",
                    "test histogram",
                    [10, 20, 15],
                    xlabels=["A", "B", "C"],
                    yaxis="value",
                    xaxis="category",
                    extra_layout={"orientation":'h'}
                )
```

I required this functionality for two reasons:

1. When comparing between experiments `report_histogram` automatically shows the bars from the different  experiments on the same graph making it much more easier to compare.
2. I have many bars (more than 20) which makes vertical orientation very hard to view and use.

As for the source of the problem, quoting **Martin.B** from the slack channel :
> since at the end this become a plotly object the extra_layout is for general purpose layout, but this specific entry is next to the data

**Comment by erezalg** [comment]
Hi @ohad-pipl,

Yeah :) There seem to be a problem with the way we handle or pass extra_layout to the plotly builder. We'll work on it and come up with a fix. We'll update you here once we have something!

**Comment by pollfly** [comment]
Hey @ohad-pipl! Just letting you know that this issue has been resolved in the recently released [v1.7.0](https://github.com/allegroai/clearml/releases/tag/v1.7.0). Let us know if there are any issues :)

**Comment by jkhenning** [comment]
Closing this issue. Please reopen if it's still relevant.
